app/components/admin/custom_fields/hierarchy/item_component.html.erb

```diff
@@ -46,10 +46,14 @@ See COPYRIGHT and LICENSE files for more details.
 
       # Actions
       item_container.with_column do
-        render(Primer::Beta::IconButton.new(scheme: :default,
-                                            icon: "kebab-horizontal",
-                                            "aria-label": I18n.t("custom_fields.admin.items.more_actions")))
+        render(Primer::Alpha::ActionMenu.new(data: { test_selector: "op-hierarchy-item--action-menu" })) do |menu|
+          menu.with_show_button(icon: "kebab-horizontal",
+                                scheme: :invisible,
+                                "aria-label": I18n.t("custom_fields.admin.items.actions"))
+          delete_item(menu)
```

Comment:
At first read I though the intention was to delete something, not to add something to the menu :shrug: 

Do we have a standard here or could we name the method something else?

Reply:
no standard, but we can definitely improve method name